docs/open-source-projects.md

```diff
@@ -240,6 +240,24 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## Boilerplates
 
+> EOS Costa Rica offers a wide variety of boilerplates for the developer community.
+
+
+<div class="table-wrapper" markdown="block" style={{overflow: 'scroll', height: '200px'}}>
+
+| Boilerplates | Description | GitHub |
+|    :----:    |   :----:    | :----: |
+| Full Stack | A fullstack project template to easily launch your project on EOSIO. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/full-stack-boilerplate)     |
+| Wax Webapp | A bare-bones react web app featuring WAX authenticators (wallets). | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/wax-webapp-boilerplate)     |
+|     Wax    | A full-stack project template to easily launch your project on Wax. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/wax-full-stack-boilerplate) |
+|   Proton   | A full-stack project template to easily launch your project on Proton. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/proton-full-stack-boilerplate) |
+|   Webapp   | A bare-bones react web app featuring UAL for EOSIO authenticators (wallets). | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/webapp-boilerplate) |
+|   IPFS   | A repository to help you get started integrating IPFS into your project. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/ipfs-boilerplate) |
+|   Backend   | NodeJS Backend + Hasura + PGDB with EOSJS and KEOSD wallet. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/backend-boilerplate) |
+|   EOS dapp   | EOSIO Application Boilerplates. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/eos-dapp-boilerplate) |
+|   EOS dapp   | EOSIO Application Boilerplates. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/eos-dapp-boilerplate) |
+
+</div>
```

Comment:
I like the solution to organize the information, It is more understandable now. I only have one comment, what if the repository link is added in the boilerplate name? The GitHub logo seems to be a little repetitive.

Reply:
@leisterfrancisco So do you think it's better to just keep it in the name only and remove the logo column or keep it all and add the link to the names?

Reply:
Remove logo column